src/storage/v2/disk/storage.cpp

```diff
@@ -90,7 +90,7 @@ bool VertexHasLabel(const Vertex &vertex, LabelId label, Transaction *transactio
   bool has_label = false;
```

Comment:
Just a note: in a separate PR taking locks on vertices and edges in disk/storage.cpp will be removed since it is not needed

Reply:
After rebase all changes to on disk have been removed